lib/TextField/TextField.js

```diff
@@ -113,9 +113,11 @@ class TextField extends React.Component {
     }
 
     setInputClassName() {
-        const { classes, iconName, clear, type, inputClassName, alignRight, hjWhitelist } = this.props;
+        const { classes, iconName, clear, type, inputClassName, alignRight, hjWhitelist, searchField } = this.props;
         let className = classes.input;
+        if (searchField) className += ' ' + classes.searchField;
         if (iconName) className += ' ' + classes.hasIcon;
+        if (searchField && iconName) className += ' ' + classes.searchFieldHasIcon;
         if (clear) className += ' ' + classes.hasClear;
         if (alignRight) className += ' ' + classes.alignRight;
         if (type == "select") className += ' ' + classes.select;
```

Comment:
Para añadir al comentario de @editron, también podría quedar mejor así, aunque creo que también podría ser bueno apegarse a lo realizado en otros archivos para mantener consistencia 🤔 

```suggestion
        if (searchField) `${className} ${classes.searchField}`;
        if (iconName) `${className} ${classes.hasIcon}`;
        if (searchField && iconName) `${className} ${classes.searchFieldHasIcon}`;
        if (clear) `${className} ${classes.hasClear}`;
        if (alignRight) `${className} ${classes.alignRight}`;
        if (type == "select") `${className} ${classes.select}`;
```

Reply:
Nada más que en tu sugerencia no se está asignando el valor a nada, tendría qué ser algo como `className = '${className} ${classes.searchField}'`, que parece mucho más extenso. Pero para ese caso vería entonces más preferible por ser más corto la opción `className += ' ${classes.searchField}'`. (puse comillas simples porque no puedo escapar en el comentario el signo de `  😅 )

Reply:
Tienes razón, necesito entrenar el notepad, no me di cuenta de eso😅